### DISCUSSION TITLE
Storage of forwarding rules

### DISCUSSION THREAD

**Discussion by ls-2018** [skupperproject/skupper#1751] [Ideas]
The current forwarding rule is stored in configmap, but it has a size limit.
When the expose rule becomes too numerous, it is no longer allowed to create.
Are there any further ideas to optimize this point?

**Comment by ted-ross** [comment]
Skupper V2 no longer uses the config-map for endpoint configuration.  It uses a custom-resource per endpoint.

**Reply by ls-2018** [reply]
When will v2 be released？
